chain/client/src/view_client.rs

```diff
@@ -140,14 +140,17 @@ impl ViewClientActor {
         })
     }
 
-    fn maybe_block_id_to_block_hash(
+    fn maybe_block_id_to_block_header(
```

Comment:
does it still have to be mutable self?

Reply:
and I'd also add the comment (explaining what we return if MaybeBlockId is None)